src/resources/i18n-target.ts

```diff
@@ -0,0 +1,34 @@
+import { customElement, autoinject } from "aurelia-framework";
+import { I18N, RelativeTime } from "aurelia-i18n";
+
+@customElement("i18n-target")
+@autoinject
+export class I18NTarget {
+    dispatchedOn = new Date(2020, 1, 10, 5, 15);
+    deliveredOn = new Date(2021, 1, 10, 5, 15);
+    myDate: Date;
+    locale: string;
+    translations: any;

```

Comment:
`{ [key: string]: string }` instead of any

Reply:
Done